components/sidebar/sidebar_channel_list/sidebar_channel_list.tsx

```diff
@@ -362,6 +365,10 @@ export default class SidebarChannelList extends React.PureComponent<Props, State
         const droppable = [...document.querySelectorAll<HTMLDivElement>('[data-rbd-droppable-id*="droppable-categories"]')];
         droppable[0].style.height = `${droppable[0].scrollHeight}px`;
 
+        if (!this.props.selectedChannelIds.find((id) => before.draggableId === id)) {
```

Comment:
I was going to suggest `some` since it returns a boolean, but it turns out they perform the same. Nothing to change here, but I thought that might be interesting to know.

Reply:
I do enjoy facts :D